.github/workflows/test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       run: go build -v .
 
     - name: TF tests
-      run: go test -v --coverprofile coverage.txt -covermode=atomic -parallel 4 $(go list ./... |grep -v 'tests')
+      run: go test -v --coverprofile coverage.txt -covermode=atomic -parallel 4 ./...
```

Comment:
Perhaps the intent of the old code was to ignore coverage of the tests themselves (or testing_helpers). Should be fine.

Reply:
The `go list` subcommand was added to avoid running the terragrunt tests in the `tests` directory since there was no way to make them work in CI.  Those tests have since been deleted, so the subcommand was effectively a no-op after that.